sleap/nn/system.py

```diff
@@ -190,28 +190,35 @@ def get_gpu_memory() -> List[int]:
     """Return list of available GPU memory.
 
     Returns:
-        List of available GPU memory with indices corresponding to GPU indices.
+        List of available GPU memory (in MiB) with indices corresponding to GPU indices.
+
+    Notes:
+        This function depends on the `nvidia-smi` system utility. If it cannot be found
+        in the `PATH`, this function returns an empty list.
     """
     command = [
         "nvidia-smi",
         "--query-gpu=memory.free",
         "--format=csv",
     ]
 
-    memory_poll = subprocess.run(command, capture_output=True)
+    try:
+        memory_poll = subprocess.run(command, capture_output=True)
+    except (subprocess.SubprocessError, FileNotFoundError):
+        return []
```

Comment:
It would be nice to have a logger print something here for when we actually find a good GPU to use or just default to 0.